astroid/nodes.py

```diff
@@ -78,6 +79,7 @@
     MatchValue,
     Name,
     NamedExpr,
+    NodeNG,
```

Comment:
A Changelog entry explaining that `NodeNG` can now be accessed through `from astroid import NodeNG` would be helpful. It could also mention that the previous import is now deprecated.

Reply:
I added a message in changelog. The previous import will be deprecated at the same time than all the other in #1057